CollAction/Views/Shared/_Footer.cshtml

```diff
@@ -59,35 +59,10 @@
             <li><a target="_blank" href="https://www.linkedin.com/company-beta/15079855/"><div><i class="fa fa-linkedin"></i></div></a></li>
         </ul>
 
-        <h5 class="header">@Localizer["Newsletter"]</h5>
-
-        <!-- Begin MailChimp Signup Form -->
-        <div id="mc_embed_signup">
-          <form action="//collaction.us14.list-manage.com/subscribe/post?u=48e9b2f8f522cf59b9d5ffa8d&amp;id=@NewsletterSubscriptionServiceOptions.Value.MailChimpNewsletterListId" method="post"
-            id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-            <div id="mc_embed_signup_scroll">
-              <div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
-              <div class="mc-field-group">
-                <label for="mce-EMAIL">Email Address  <span class="asterisk">*</span></label>
-                <input type="email" placeholder="E-mail" value="" name="EMAIL" class="required email" id="mce-EMAIL">
-                <i class="fa fa-envelope-o email-icon"></i>
-              </div>
-              <div id="mce-responses" class="clear">
-                <div class="response" id="mce-error-response" style="display:none"></div>
-                <div class="response" id="mce-success-response" style="display:none"></div>
-              </div>
-              <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-              <div style="position: absolute; left: -5000px;" aria-hidden="true">
-                <input type="text" place-holder="E-Mail" name="b_48e9b2f8f522cf59b9d5ffa8d_@NewsletterSubscriptionServiceOptions.Value.MailChimpNewsletterListId" tabindex="-1" value="">
-              </div>
-              <div class="clear">
-                <input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
-              </div>
-            </div>
-          </form>
-        </div>
-        <!--End mc_embed_signup-->
+        <h5 class="header">@Localizer["Sign up to our newsletter"]</h5>
 
+        <!-- Mailchimp signup form -->
+        <div class="newsletter-signup" data-u="48e9b2f8f522cf59b9d5ffa8d" data-list-id="@NewsletterSubscriptionServiceOptions.Value.MailChimpNewsletterListId"></div>
```

Comment:
48e9b2f8f522cf59b9d5ffa8d probably needs to be fed from a configuration value, just like @NewsletterSubscriptionServiceOptions.Value.MailChimpNewsletterListId

Reply:
I added it as a new variable @NewsletterSubscriptionServiceOptions.Value.MailChimpUserId